libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1227,7 +1227,7 @@ void RCOutput::timer_tick(uint32_t time_out_us)
     }
 
     // if we have enough time left send out LED data
-    if (serial_led_pending && (time_out_us > (AP_HAL::micros() + (_dshot_period_us >> 1)))) {
```

Comment:
what is the issue there ? possible uint32_t overflow and implicit cast to uint64_t ?